reference/sql/statements/flashback-table.md

```diff
@@ -0,0 +1,84 @@
+---
+title: FLASHBACK TABLE
+summary: Learn how to recover tables using the `FLASHBACK TABLE` statement.
+category: reference
+---
+
+# FLASHBACK TABLE
+
+The `FLASHBACK TABLE` statement is introduced in TiDB 4.0. You can use the `FLASHBACK TABLE` statement to restore the tables and data dropped by `DROP` or `TRUNCATE` within the Garbage Collection (GC) lifetime.
```

Comment:
```suggestion
The `FLASHBACK TABLE` syntax is introduced since TiDB 4.0. You can use the `FLASHBACK TABLE` statement to restore the tables and data dropped by the `DROP` or `TRUNCATE` operation within the Garbage Collection (GC) lifetime.
```